torch/_dynamo/symbolic_convert.py

```diff
@@ -1472,6 +1475,9 @@ def MATCH_KEYS(self, inst):
             if sys.version_info < (3, 11):
                 self.push(ConstantVariable(False))
 
+    def LOAD_ASSERTION_ERROR(self, inst):
```

Comment:
If we have this here, why do we need the previous statement? Shouldn't it error here when we encounter instruction ASSERTION_ERROR.

Reply:
```LOAD_ASSERTION_ERROR``` is for python 3.9+, ```LOAD_GLOBAL              0 (Assertion type)``` is for python 3.8(-)

Reply:
Oh right.